meerk40t/core/elements.py

```diff
@@ -223,30 +223,30 @@ def plugin(kernel, lifecycle=None):
                 "page": "Classification",
                 "section": "_90_Auto-Generation",
             },
-            {
```

Comment:
This rather large commenting out doesn't seem to be related or reflected in the PR title and seems like it might want to be a different PR.

Reply:
Agree